arch/arm/src/stm32h7/stm32_allocateheap.c

```diff
@@ -246,6 +249,14 @@ void up_allocate_heap(FAR void **heap_start, size_t *heap_size)
 
   up_heap_color(*heap_start, *heap_size);
 #endif
+
+#if defined(CONFIG_DEBUG_FEATURES)
+
+  /* Display memory ranges to help debugging */
+
+  _info("%uKb of SRAM at %p\n", *heap_size / 1024, *heap_start);
```

Comment:
In practice the underscore version is not use directly. see debug.h minfo() would be used here.
```suggestion
  minfo("%uKb of SRAM at %p\n", *heap_size / 1024, *heap_start);
```

Reply:
@anpaza let me know if you are OK with this change and I will commit and merge it.

Reply:
Sorry for late answer.
Changed as you suggested.

Reply:
Tried compiling all stm32h7 targets:
  stm32h747i-disco:nsh
  nucleo-h743zi:nsh
  nucleo-h743zi:nxlines_oled
  nucleo-h743zi:pwm
without FMC they compile fine, with FMC enable they won't compile because their board.h does not define the respective macros, needed to configure FMC for their board (for example, BOARD_FMC_GPIO_CONFIGS should list the GPIOs used for external memories, without it FMC has no sense).

Reply:
So the `correct and best` thing to do is to look and see if the HW has an SDRAM in it. Then add a config and update board.h for thoses ones.  It if does not then there is no action needed.

The issue is that, if you do not have the HW you can not verify it. But you are the `expert` on this at this point in time and can share that expertise in a board config so that when some does try to use it, they are closer to making it work that staring from scratch.

Reply:
In most cases, _info or minfo will not work when called from up_allocateheap().  up_allocateheap() is called earlier in the initialization sequence than is the logic that initialzes the syslog output.  Do you actually see output?